content/release-10-17-0/oee-10-17-0.md

```diff
@@ -0,0 +1,154 @@
+---
+weight: 95
+title: OEE App
+layout: bundle
+---
+
+### Improvements
+
+
+<table ><colgroup>
+<col style="width: 17%;">
+<col style="width:57%;">
+<col style="width: 11%;">
+<col style="width: 15%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Administration</td>
+<td>This release of the Cumulocity IoT OEE Application adds the possibility to calculate the OEE value based on defined shift plans. A new and improved user interface is available to allow OEE Administrators to add, edit and remove shift plans.</td>
+<td>COE-549</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Product experience</td>
+<td>Updated the WebSDK to 10.17 in order to adopt the new Cumulocity UI.</td>
+<td>COE-445</td>
+<td>UI</td>
+</tr>
+
+</tbody></table>
+
+### Fixes
+
+<table >
+<colgroup>
+<col style="width: 17%;">
+<col style="width:57%;">
+<col style="width: 11%;">
+<col style="width: 15%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Product experience</td>
+<td>Previously, locations given in the Filter Location dropdown in the Profiles page were not sorted. They are now sorted alpabetically in order to improve findability. </td>
+<td>COE-884</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>Improve the behaviour of the Microservices during startup. Includes fixes for an issue that caused "Permission Denied" messages in certain configurations.</td>
+<td>COE-857</td>
+<td>Microservice</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>Improvements to the handling of the log files created by the microservice.</td>
+<td>COE-777</td>
+<td>Microservice</td>
+</tr>
+
+<tr>
+<td>Product experience</td>
+<td>Removed obsolete Section View from the Machine Dashboard.</td>
+<td>COE-735</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Product experience</td>
+<td>Improved the rendering behaviour of webpages when the user's language is set to Japanese.</td>
+<td>COE-732</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Product experience</td>
+<td>Add more translated strings to improve localisation.</td>
+<td>COE-681</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Product experience</td>
+<td>The behaviour of the Organisation Overview page was updated. It will now handle cases where OEE profiles are missing or removed in a more user-friendly and obvious way.</td>
```

Comment:
```suggestion
<td>The behaviour of the <b>Organization overview</b> page was updated. It now handles cases where OEE profiles are missing or removed in a more user-friendly and obvious way.</td>
```

Reply:
Is that the name of the page? Isn´t it just called "Overview"?

Reply:
Yes. ln the UI, it's always called Overview.

Reply:
```suggestion
<td>The behaviour of the <b>Overview</b> page was updated. It now handles cases where OEE profiles are missing or removed in a more user-friendly and obvious way.</td>
```